azurerm/resource_arm_application_gateway.go

```diff
@@ -865,6 +873,7 @@ func resourceArmApplicationGatewayRead(d *schema.ResourceData, meta interface{})
 
 	d.Set("name", applicationGateway.Name)
 	d.Set("resource_group_name", id.ResourceGroup)
+	d.Set("enable_http2", applicationGateway.EnableHTTP2)
```

Comment:
post-reviewing this: we want to be access this using the `properties.EnableHTTP2` field rather than the `applicationGateway.EnableHTTP2` field which doesn't necessarily map the API response - will push a commit for this

Reply:
all good, I only spotted it whilst passing through 😄 - there's a PR here: https://github.com/terraform-providers/terraform-provider-azurerm/pull/2745